optical_load.ipynb

```diff
@@ -2,7 +2,25 @@
  "cells": [
```

Comment:
all these changes are a bit confusing because they are in the notebook, this is why it is probably easier to just collaborate on the python document itself not in the notebook

Reply:
I agree. It adds information to the document for every little thing